cgsmiles/resolve.py

```diff
@@ -189,6 +182,10 @@ def edges_from_bonding_descrpt(self):
                 # unless they are specifically annotated
                 order = int(bonding[0][-1])
                 self.molecule.add_edge(edge[0], edge[1], bonding=bonding, order=order)
+                if self.all_atom:
+                    for edge_node in edge:
+                        if self.molecule.nodes[edge_node]['element'] != 'H':
+                            self.molecule.nodes[edge_node]['hcount'] -= 1
```

Comment:
```suggestion
                            self.molecule.nodes[edge_node]['hcount'] -= order
```
?

Reply:
this works is the short answer; having the order there does not presumably due to aromatic bond orders